lib/remix.ex

```diff
@@ -1,7 +1,3 @@
 defmodule RemixedRemix do
   @moduledoc false
-
-  def hello() do
```

Comment:
Although I have 0 tests in the project currently (it's part of my roadmap to add tests to all of my projects, this is just low on the list 😅), this is still being referenced in the test file. Make sure to remove the references to this function, (we will add tests later on)